include/deal.II/lac/solver_qmrs.h

```diff
@@ -205,67 +239,56 @@ class SolverQMRS : public Solver<VectorType>
 
 template <class VectorType>
 SolverQMRS<VectorType>::IterationResult::IterationResult (const SolverControl::State state,
-                                                          const double               last_residual)
+                                                          const double last_residual)
   :
   state (state),
   last_residual (last_residual)
-{}
-
+{
+}
 
 template <class VectorType>
-SolverQMRS<VectorType>::SolverQMRS (SolverControl            &cn,
+SolverQMRS<VectorType>::SolverQMRS (SolverControl &cn,
                                     VectorMemory<VectorType> &mem,
-                                    const AdditionalData     &data)
+                                    const AdditionalData &data)
   :
-  Solver<VectorType>(cn,mem),
-  additional_data(data)
-{}
-
-
+  Solver<VectorType> (cn, mem),
+  additional_data (data)
+{
+}
 
 template <class VectorType>
-SolverQMRS<VectorType>::SolverQMRS(SolverControl        &cn,
-                                   const AdditionalData &data)
+SolverQMRS<VectorType>::SolverQMRS (SolverControl &cn,
+                                    const AdditionalData &data)
   :
-  Solver<VectorType>(cn),
-  additional_data(data)
-{}
-
-
-
-template <class VectorType>
-double
-SolverQMRS<VectorType>::criterion()
+  Solver<VectorType> (cn),
+  additional_data (data)
 {
-  return std::sqrt(res2);
 }
 
-
-
 template <class VectorType>
 void
-SolverQMRS<VectorType>::print_vectors(const unsigned int,
-                                      const VectorType &,
-                                      const VectorType &,
-                                      const VectorType &) const
-{}
-
-
+SolverQMRS<VectorType>::print_vectors (const unsigned int,
+                                       const VectorType &,
+                                       const VectorType &,
+                                       const VectorType &) const
+{
+}
 
 template <class VectorType>
 template <typename MatrixType, typename PreconditionerType>
 void
-SolverQMRS<VectorType>::solve (const MatrixType         &A,
-                               VectorType               &x,
-                               const VectorType         &b,
+SolverQMRS<VectorType>::solve (const MatrixType &A,
+                               VectorType &x,
+                               const VectorType &b,
                                const PreconditionerType &preconditioner)
 {
-  LogStream::Prefix prefix("QMRS");
+  LogStream::Prefix prefix("SQMR"); //TODO Renaming issue
```

Comment:
The change is ok, but please remove the `TODO` marker

Reply:
Okay, done.